comp/vision/gamepiece_finder24.py

```diff
@@ -191,13 +197,14 @@ def find_object(self, img_yuv):
 
             cX = int(mmnts["m10"] / mmnts["m00"])
             cY = int(mmnts["m01"] / mmnts["m00"])
-
+            xAngle = ((cY - (self.height/2))/self.height)*self.vertFOV
+            yAngle = ((cX - (self.width/2))/self.width)*self.horzFOV
```

Comment:
i think here you're trying to represent "yaw" which is positive-right in camera coordinates